README.md

```diff
@@ -0,0 +1,58 @@
+# Aligent Magento Unit Testing Pipe
+
+A bitbucket pipe for running Magento unit tests
+
+It is designed to be run in parallell, so you can leverage bitbucket parallel steps. [Example pipeline](#example-pipeline)
```

Comment:
this is intended? it is still true though :thinking:

Reply:
Well, not in parallel with itself, but with other steps, sure.

Reply:
makes sense to be parallel with perhaps multiple different groups? i guess that encourages us to write some pretty hefty tests :weight_lifting: :white_check_mark: